### PR TITLE
Update website for GRC

### PR DESCRIPTION
**Pull request by dockerymick**
Signed-off-by: Mikela Jackson <mdockery@redhat.com>

### CODE REVIEW

**Comment by dockerymick** [comment]
Since @ycao56 approved the previous PR, Mike are you able to merge this? @mikeshng 

 previous PR

File: content/getting-started/install-policy-framework.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Install Policy Framework
 weight: 4
 ---
 
-After cluster manager and klusterlet are installed, you could install the policy framework components to the hub and the managed clusters.
+After the hub and klusterlet are installed, you can install the policy framework components to the hub and the managed clusters.
```
Comment (mikeshng) [inline content/getting-started/install-policy-framework.md:6]: `After cluster manager and klusterlet are installed`

we changed some of the `hub` wording to `cluster manager`

File: content/getting-started/install-policy-framework.md [hunk 2]
```diff
@@ -13,61 +13,67 @@ After cluster manager and klusterlet are installed, you could install the policy
...
+* Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [`kustomize`](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
 
-Prepare one Kubernetes cluster to function as the hub. For example, use [kind](https://kind.sigs.k8s.io/docs/user/quick-start) to create a hub cluster. To use kind, you will need [docker](https://docs.docker.com/get-started) installed and running.
+* Ensure [Golang](https://golang.org/doc/install) is installed, if you are planning to install from the source.
 
-Ensure the open-cluster-management cluster manager is installed. See [Install Cluster Manager](../install-cluster-manager) for more information.
+* Prepare one Kubernetes cluster to function as the hub cluster. For example, use [kind](https://kind.sigs.k8s.io/docs/user/quick-start) to create a hub cluster. To use kind, you must install and run [Docker](https://docs.docker.com/get-started).
 
-Ensure the open-cluster-management klusterlet is installed. See [Install Klusterlet](../register-cluster) for more information.
+* Ensure the `open-cluster-management` _hub_ is installed. See [Install Hub](install-hub.md) for more information.
```
Comment (mikeshng) [inline content/getting-started/install-policy-framework.md:24]: Ensure the open-cluster-management cluster manager is installed. See [Install Cluster Manager](../install-cluster-manager)

File: content/getting-started/install-policy-framework.md [hunk 3]
```diff
@@ -13,61 +13,67 @@ After cluster manager and klusterlet are installed, you could install the policy
...
 
-Ensure the open-cluster-management application management is installed. See [Install Application management](../install-application) for more information.
+* Ensure the `open-cluster-management` _klusterlet_ is installed. See [Install Klusterlet](register-cluster.md) for more information.
```
Comment (mikeshng) [inline content/getting-started/install-policy-framework.md:26]: fix the link here it should be: 
See [Install Klusterlet](../register-cluster) for more information.

File: content/getting-started/install-policy-framework.md [hunk 4]
```diff
@@ -13,61 +13,67 @@ After cluster manager and klusterlet are installed, you could install the policy
...
+
+* Ensure the `open-cluster-management` _Application_ is installed. See [Install Application management](install-application.md) for more information.
```
Comment (mikeshng) [inline content/getting-started/install-policy-framework.md:28]: fix link here:

 See [Install Application management](../install-application) for more information.

**Review by mikeshng — CHANGES_REQUESTED** [review]
Just some small fixes needed. Thanks @dockerymick

**Review by mikeshng — APPROVED** [review]
/approved /lgtm
thanks

**Pull request merged**